docs/containerd-env-setup.md

```diff
@@ -132,9 +134,9 @@ sudo docker run -d --restart=always -p 5000:5000 registry
 
 ## Convert an Image to Nydus Format
 
-Currently, nydus image must be created by converting from an existed OCI or docker v2 image.
+Nydus image can be created by converting from an existing OCI or docker v2 image stored in container registry or directly built from Dockerfile(with Buildkit)
```

Comment:
Can we give the buildkit doc link: https://github.com/moby/buildkit/blob/master/docs/nydus.md